### PR TITLE
Actions testing

### PR DESCRIPTION
**Pull request by ericberglund117**
### What does this PR do?
- create separate testing files for each action in our index.js in the actions directory
- pass the addID, isLoading, hasError, and setCountries actions test

### Where should the reviewer start?
- actions directory and look at each test
- run npm test

### How should this be manually tested?
-npm test

### Any background context you want to provide?
- Pretty simple and straightforward testing
-Needed to test if the returned output is what we expect based on the inputs

### CODE REVIEW

File: src/actions/hasErrored.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import * as actions from '../actions'
```
Comment (taylorjohnson141) [inline src/actions/hasErrored.test.js:1]: The star import looks reallypowerful!

File: src/actions/setCountries.test.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    ]
+
+    const expectedAction = {
+      type: 'SET_COUNTRIES',
+      countries: countries
+    }
+
+    const result = actions.setCountries(countries);
+
+    expect(result).toEqual(expectedAction)
```
Comment (taylorjohnson141) [inline src/actions/setCountries.test.js:103]: This all looks really good!

**Review by taylorjohnson141 — APPROVED** [review]

File: src/actions/hasErrored.test.js (hunk 1 above)
Comment (timkeresey) [reply src/actions/hasErrored.test.js:1]: Yeah that is a good tool to use from now on.

**Review by timkeresey — COMMENTED** [review]
Action testing is straightforward so this should be the extent of what we need.

**Pull request merged**